{
  "WorkItem": {
    "AffectedComponent": {
      "Name": "",
      "DisplayName": ""
    },
    "ClosedComment": "This has been fixed for the 0.9 release.",
    "ClosedDate": "2013-04-10T16:37:19.667-07:00",
    "CommentCount": 4,
    "Custom": null,
    "Description": "Hi, this bug has been around for a while - just finally figured out how to reproduce it:\r\n\r\n```\r\n// If this module declaration is removed then the error goes away\r\nmodule a {\r\n}\r\n\r\nmodule a {\r\n\t\t\r\n    class Local {\r\n    }\r\n\r\n    export class Global {\r\n\t\t\r\n       private local : Local = undefined;\r\n\t   // Error: The name 'Local' does not exist in the current scope\r\n    }\r\n}\r\n```",
    "LastUpdatedDate": "2014-01-10T10:34:33.287-08:00",
    "PlannedForRelease": "",
    "ReleaseVisibleToPublic": false,
    "Priority": {
      "Name": "Low",
      "Severity": 50,
      "Id": 1
    },
    "ProjectName": "typescript",
    "ReportedDate": "2013-01-24T02:16:58.887-08:00",
    "Status": {
      "Name": "Closed",
      "Id": 4
    },
    "ReasonClosed": {
      "Name": "Fixed"
    },
    "Summary": "Spurious \"The name does not exist in the current scope\"",
    "Type": {
      "Name": "Issue",
      "Id": 3
    },
    "VoteCount": 6,
    "Id": 641
  },
  "FileAttachments": [],
  "Comments": [
    {
      "Message": "I have exactly the same issue but the error which I get is: \n\"The symbol 'SomeSymbolName' is not visible at this point\"\n\nThe error message is different since I use the import directive.\nFor me that specific error appeared after migration to 0.8.2 from 0.8.1\n\nIt may somehow also be related to the issue which I have reported under the id 616 (but I am no 100% sure)\n\nRegards\nPiotr Spikowski",
      "PostedDate": "2013-01-24T10:36:41.75-08:00",
      "Id": 87155
    },
    {
      "Message": "Marking 'active'",
      "PostedDate": "2013-01-28T09:44:30.73-08:00",
      "Id": 87520
    },
    {
      "Message": "I'm having a similar and or same issue but I haven't managed to isolate it.\n",
      "PostedDate": "2013-02-15T12:54:57.637-08:00",
      "Id": 90505
    },
    {
      "Message": "I have the same issue but I have a module of the same name in a separate file and I <reference/> it. If I remove the reference, the error disappears.\n\n-Nodir",
      "PostedDate": "2013-03-12T12:32:31.94-07:00",
      "Id": 93063
    }
  ]
}